Source/ACE/Entity/Player.cs

```diff
@@ -473,10 +259,32 @@ public void Load(AceCharacter character)
 
             IsOnline = true;
 
-            // Character.AnimationOverrides.ForEach(ao => this.ModelData.AddModel(ao.Index, ao.AnimationId));
-            // Character.TextureOverrides.ForEach(to => this.ModelData.AddTexture(to.Index, to.OldId, to.NewId));
-            // Character.PaletteOverrides.ForEach(po => this.ModelData.AddPalette(po.SubPaletteId, po.Offset, po.Length));
+            PhysicsData.MTableResourceId = Character.MotionTableId;
+            PhysicsData.Stable = Character.SoundTableId;
+            PhysicsData.Petable = Character.PhysicsTableId;
+            PhysicsData.CSetup = Character.ModelTableId;
+
+            ContainerCapacity = 7;
+
+            if (Character.DefaultScale != null)
+                PhysicsData.ObjScale = Character.DefaultScale;
+
+            // Hair/head
```

Comment:
yea - we were discussing that the other day
Right now we reload that base each time.     we are going to have to decide if we want to store your base undressed appearance and build up from there - which I think is a good way to do it
that way logging in is behind the scenes just equipping your items
and your appearance is the addition of your armor ordered by priority

I will add a card on Trello.